src/caduceus/mocks_test.go

```diff
@@ -117,6 +123,11 @@ func (m *mockGauge) Set(value float64) {
 }
 
 func (m *mockGauge) With(labelValues ...string) metrics.Gauge {
+	for _, v := range labelValues {
+		if !utf8.ValidString(v) {
+			panic("not UTF-8")
+		}
+	}
 	args := m.Called(labelValues)
```

Comment:
Can this line be before the labelValues check?

Reply:
yes - good catch

Reply:
I flipped it back so we don't need to add non UTF-8 values to the tests... we will clean this up when we do some pretty serious house cleaning on the code.